.github/CONTRIBUTING.md

```diff
@@ -96,7 +97,21 @@ If you are adding new assets that are not already explicitly licensed under CC-B
 
 If at any point you are confused or unsure of an asset's license or our policy, ask a Maintainer to help you.
 
-## 7. Banned Content
+## 7. Pull Request Standards
+
+You should complete the pull-request template in its entirety.
+
+Any pull-request that does not adequately complete the provided template may be closed or marked 'do not merge' by maintainers.
+ - Balance changes must indicate that they affect the balance of the game.
+ - Changes must be documented in their entirety including the extent of their effects. (For example, if you change it so all mobs are half speed, don't label the PR as 'monkeys now move twice as slow'). Failing to document the full extent of the changes may result in a repo-ban if the intent of hiding changes is seen as malicious.
+ - The section labeled 'about this pull request' should state the changes that the pull-request makes.
```

Comment:
I feel like this point is too terse/unverbose, it might as well describe the changelog. I'd suggest either getting rid of it, or something more verbose like

```suggestion
 - The section labeled 'about this pull request' should describe the pull request in detail. This includes the change being made from the player's perspective, any important details about how it was implemented, as well as background on the pull request, such as the source of what might be ported or issues that are being fixed. Note that it should not include images and videos, there is a dedicated section for those.
```

Reply:
About this pull request should allow for images and videos in it if the images and videos show the changes, I think thats too strong.

Imo the 'about this pull request' section should strictly be the what, the changes how they are in an objective manner, with the reason for implementing them going in the 'Why its good for the game section'.

The main reason for this bit is because all the other sections have a quick description too, about this pull request is pretty self-explanatory.

Reply:
> About this pull request should allow for images and videos in it if the images and videos show the changes, I think thats too strong.

This is a matter of personal opinion, but I was supportive of adding the `Screenshots&Videos` collapsible specifically because those tend to take up a lot of space, making browsing PRs painful. That is, it was my understanding that the collapsible is where screenshots and videos should go in general, and I had the impression we were in agreement about it.

> Imo the 'about this pull request' section should strictly be the what, the changes how they are in an objective manner, with the reason for implementing them going in the 'Why its good for the game section'.

I mostly agree, and I think the suggested changes do not contradict anything you said in this reply. Might be a case of bad wording, but for clarity, what I suggested it should be is:
- The actual changes, based on how they affect the users, in detail.
- Important implementation details. In cases of refactors/new systems/maintenance PRs, this would be the bulk of it, since they won't affect users much. In other PRs, this would only be needed when big/risky changes are being made, or implementation is changed in a way that's important to future coders.
- The background on the PR. It's not meant to be the reason for why it's good for the game, but rather elaboration on how the PR came to be. This is meant to be mainly PRs from other codebases being ported, Issues that are being fixed, previous PRs from this codebase that are being addressed in some ways. That said, if the author wants to cite other background, such as previous discussion about the matter, I feel like this is the good place to do it.

I also want to note that I see the "Why it's good for the game" as being a more strict section - to me, it shouldn't have vague elaboration, but specific reasoning for why it's an improvement. Because of this, anything like points to discuss further that are up to debate would be something to put in "About this pull request".

Reply:
```suggestion
 - The section labeled 'about this pull request' should describe the pull request's changes in detail. This includes the changes being made, any important details about how it was implemented, the issues it closes and links to any other pull requests if code is being ported from another codebase.
```

Reply:
We're not animals- use the Oxford comma.

Reply:
what is an oxford comma?

Reply:
ew that is just wrong

Reply:
You're objectively wrong. Add the comma, it's far more readable with it.

Reply:
Fine

Reply:
That good?